syntax = "proto3";
package pool;

import "common.proto";


service ModelDivide {
    rpc divide_model(PartitionRequest) returns (PartitionResponse);
}




message PartitionRequest {
    common.ModelId model_id = 1;
    string solved_graph = 2;
}

message PartitionResponse {
}